test/distributed/org/apache/cassandra/distributed/test/UpgradeSSTablesTest.java

```diff
@@ -59,8 +59,8 @@ public void upgradeSSTablesInterruptsOngoingCompaction() throws Throwable
             cluster.get(1).acceptsOnInstance((String ks) -> {
                 ColumnFamilyStore cfs = Keyspace.open(ks).getColumnFamilyStore("tbl");
                 cfs.disableAutoCompaction();
-                CompactionManager.instance.setMaximumCompactorThreads(1);
                 CompactionManager.instance.setCoreCompactorThreads(1);
+                CompactionManager.instance.setMaximumCompactorThreads(1);
```

Comment:
Why this reordering?

Reply:
Because the previous order was invalid (if the core is e.g. 4 and you set the max to 1, it's an invalid operation). Reducing the core first is the correct way to do it.